src/bootstrap/lib.rs

```diff
@@ -1209,6 +1209,10 @@ impl Build {
             && (self.hosts.iter().any(|h| *h == target) || target == self.build)
     }
 
```

Comment:
```suggestion

    /// Whether to force the use of stage 2 compiler in any case
```

I think starting to add doc-comments to all new things will help in the future.